packages/logs/src/boot/logs.ts

```diff
@@ -128,5 +128,18 @@ interface Rum {
 
 function getRUMInternalContext(startTime?: number): Context | undefined {
   const rum = (window as any).DD_RUM as Rum
-  return rum && rum.getInternalContext ? rum.getInternalContext(startTime) : undefined
+  const context = rum && rum.getInternalContext ? rum.getInternalContext(startTime) : undefined
+  if (!context || !context.session) {
+    return context
+  }
+  // TODO settle on integration strategy
+  // tslint:disable:no-unsafe-any
+  const v2context = context as any
+  v2context.session_id = v2context.session.id
+  v2context.application_id = v2context.application.id
+  if (v2context.action) {
+    v2context.user_action = { id: v2context.action.id }
+  }
```

Comment:
Are we sure that sending merged V1 & V2 format won't cause any issue in logs/trace?

Reply:
only view.id is used by app queries on logs or trace and it has not changed.
keeping v1 attributes is only to not break potential dashboards, monitors, etc... while we figure out what we want to do here. Adding v2 attributes should not break anything.
It will stay experimental as long as the strategy is not decided.